### PR TITLE
fix(DB/Loot): Fel Creep requires Cenarion Beacon to drop from Felwood herbs

### LINKED ISSUE #25734: Felwood/Herbalism - Felcreep dropping from herbing nodes despite not fulfilling condition
### Triage

Issue linked from CC:  https://github.com/chromiecraft/chromiecraft/issues/9317

Triage Notes: 

.level 79
.go c id 3013
target self
.modify money 500000
Learn herbalism
Target self before all .setskill commands
.setskill 182 75
Learn next level herbalism
.setskill 182 150
Learn next level herbalism
.setskill 182 225
Learn next level herbalism
.setskill 182 300
.tele felwood

Get picking herbs! Eventually, even though you dont have the quest or the cenarion beacon, you will get fel creep.

https://github.com/user-attachments/assets/a30baf95-03fc-46b7-aacf-9b8f86af6525

As seen here in this video on AC locally. This character had never interacted with the quest giver, and definitely didn't have the beacon as seen in the bags and indeed never did have it.

https://www.wowhead.com/wotlk/item=11514/fel-creep#comments:id=8808:reply=3943
Wowhead comment mentioning the requirement of the beacon

And of course 
https://wowgaming.altervista.org/aowow/?item=11511#condition-for
This condition is not being met.

Original Post Below: 

---
### Current Behaviour

Fel Creep (https://wowgaming.altervista.org/aowow/?item=11514) is gathered from Felwood herbs without a Cenarion Beacon (https://wowgaming.altervista.org/aowow/?item=11511) in the player's inventory.

### Expected Behaviour

Fel Creep should not be gatherable from Felwood herbs unless a Cenarion Beacon is present in the player's inventory.

### Source

https://wowgaming.altervista.org/aowow/?item=11511#condition-for 

Image

### Steps to reproduce the problem

1. Complete the quest "Cleansing Felwood"
2. Receive Cenarion Beacon from the NPC (Alliance: [Arathandris Silversky (https://www.wowhead.com/wotlk/npc=9528) or Horde: [Maybess Riverbreeze](https://www.wowhead.com/wotlk/npc=9529#00zc)
3. Delete the Cenarion Beacon from the player inventory.
4. Gather Herbs in Felwood to receive Fel Creep

### Extra Notes

_No response_

### Customizations

_No response_

### AC rev. hash/commit

https://github.com/chromiecraft/azerothcore-wotlk/commit/ddbecc2badfcf661be23caa2f5d81bede09ff0a4

### Operating system

Ubuntu 22.04

### Modules

- [mod-1v1-arena](https://github.com/azerothcore/mod-1v1-arena)
- [mod-ah-bot](https://github.com/azerothcore/mod-ah-bot)
- [mod-anticheat](https://github.com/azerothcore/mod-anticheat)
- [mod-arena-3v3-solo-queue](https://github.com/azerothcore/mod-arena-3v3-solo-queue)
- [mod-arena-replay](https://github.com/azerothcore/mod-arena-replay)
- [mod-bg-item-reward](https://github.com/azerothcore/mod-bg-item-reward)
- [mod-breaking-news-override](https://github.com/azerothcore/mod-breaking-news-override)
- [mod-cfbg](https://github.com/azerothcore/mod-cfbg)
- [mod-chat-transmitter](https://github.com/azerothcore/mod-chat-transmitter)
- [mod-chromie-xp](https://github.com/azerothcore/mod-chromie-xp)
- [mod-chromiecraft-smartstone](https://github.com/chromiecraft/mod-chromiecraft-smartstone)
- [mod-costumes](https://github.com/azerothcore/mod-costumes)
- [mod-cta-switch](https://github.com/azerothcore/mod-cta-switch)
- [mod-desertion-warnings](https://github.com/azerothcore/mod-desertion-warnings)
- [mod-dmf-switch](https://github.com/azerothcore/mod-dmf-switch)
- [mod-duel-reset](https://github.com/azerothcore/mod-duel-reset)
- [mod-eluna](https://github.com/azerothcore/mod-eluna)
- [mod-evade-logging](https://github.com/heyitsbench/mod-evade-logging)
- [mod-instanced-worldbosses](https://github.com/nyeriah/mod-instanced-worldbosses)
- [mod-ip-tracker](https://github.com/azerothcore/mod-ip-tracker)
- [mod-low-level-arena](https://github.com/azerothcore/mod-low-level-arena)
- [mod-low-level-rbg](https://github.com/azerothcore/mod-low-level-rbg)
- [mod-multi-client-check](https://github.com/azerothcore/mod-multi-client-check)
- [mod-npc-spectator](https://github.com/gozzim/mod-npc-spectator)
- [mod-progression-system](https://github.com/azerothcore/mod-progression-system)
- [mod-pvp-quests](https://github.com/azerothcore/mod-pvp-quests)
- [mod-pvp-titles](https://github.com/azerothcore/mod-pvp-titles)
- [mod-pvpstats-announcer](https://github.com/azerothcore/mod-pvpstats-announcer)
- [mod-queue-list-cache](https://github.com/azerothcore/mod-queue-list-cache)
- [mod-rdf-expansion](https://github.com/azerothcore/mod-rdf-expansion)
- [mod-resurrection-scroll](https://github.com/azerothcore/mod-resurrection-scroll)
- [mod-transmog](https://github.com/azerothcore/mod-transmog)
- [mod-weekend-xp](https://github.com/azerothcore/mod-weekend-xp)
- [mod-zone-difficulty](https://github.com/azerothcore/mod-zone-difficulty)
- [lua-CarbonCopy](https://github.com/55Honey/Acore_CarbonCopy)
- [lua-eventScripts](https://github.com/55Honey/Acore_eventScripts)
- [lua-ExchangeNpc](https://github.com/55Honey/Acore_ExchangeNpc)
- [lua-LevelUpReward](https://github.com/55Honey/Acore_LevelUpReward)
- [lua-RecruitAFriend](https://github.com/55Honey/Acore_RecruitAFriend)
- [lua-SendAndBind](https://github.com/55Honey/Acore_SendAndBind)
- [lua-TempAnnouncements](https://github.com/55Honey/Acore_TempAnnouncements)
- [lua-Zonecheck](https://github.com/55Honey/acore_Zonecheck)

### Server

ChromieCraft

### PR DESCRIPTION
**Pull request by lilliandris**
Fel Creep (11514) was dropping from Felwood herb nodes without
the player having a Cenarion Beacon (11511) in their inventory.

Added loot conditions to all affected gameobject loot entries
so Fel Creep only drops when the player has the Cenarion Beacon.

## Changes Proposed:
This PR proposes changes to:
-  [ ] Core (units, players, creatures, game systems).
-  [ ] Scripts (bosses, spell scripts, creature scripts).
-  [x] Database (SAI, creatures, etc).

### AI-assisted Pull Requests
- [x] AI tools (e.g. ChatGPT, Claude, or similar) were used entirely or partially in preparing this pull request. Please specify which tools were used, if any.

Claude Sonnet 4.6 - Max (Anthropic) was used to assist in preparing this pull request.

## Issues Addressed:
- Closes #25734

## SOURCE:
The changes have been validated through:
- [ ] Live research (checked on live servers, e.g Classic WotLK, Retail, etc.)
- [ ] Sniffs (remember to share them with the open source community!)
- [x] Video evidence, knowledge databases or other public sources (e.g forums, Wowhead, etc.)
  Wowhead comment (https://www.wowhead.com/wotlk/item=11514/fel-creep#comments:id=8808:reply=3943) and item condition source (https://wowgaming.altervista.org/aowow/?item=11511#condition-for) confirm Cenarion Beacon is required.
- [ ] The changes promoted by this pull request come partially or entirely from another project (cherry-pick).

## Tests Performed:
- [x] Tested in-game by the author.
- [ ] Tested in-game by other community members/someone else other than the author/has been live on production servers.
- [ ] This pull request requires further testing and may have edge cases to be tested.

## How to Test the Changes:
- [x] This pull request can be tested by following the reproduction steps provided in the linked issue
- [ ] This pull request requires further testing.

1. Create a character with Herbalism skill 300
2. `.tele felwood`
3. Gather Felwood herbs without Cenarion Beacon — verify Fel Creep does NOT drop
4. Obtain Cenarion Beacon (item 11511) and gather herbs again
5. Verify Fel Creep now drops correctly

### CODE REVIEW

File: data/sql/updates/pending_db_world/rev_1781654104088564177.sql [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+(4, 13967, 11514, 2, 11511, 1,
+    'Golden Sansam - Fel Creep requires Cenarion Beacon'),
+(4, 13968, 11514, 2, 11511, 1,
+    'Dreamfoil - Fel Creep requires Cenarion Beacon'),
+(4, 13969, 11514, 2, 11511, 1,
+    'Mountain Silversage - Fel Creep requires Cenarion Beacon'),
+(4, 13970, 11514, 2, 11511, 1,
+    'Arthas Tears - Fel Creep requires Cenarion Beacon'),
+(4, 13971, 11514, 2, 11511, 1,
+    'Plaguebloom - Fel Creep requires Cenarion Beacon');
```
[the remark this answers is not in the document]
Comment (lilliandris) [reply data/sql/updates/pending_db_world/rev_1781654104088564177.sql]: AzerothCore's SQL update runner executes each file atomically, 
so an explicit transaction is not needed here.

File: data/sql/updates/pending_db_world/rev_1781654104088564177.sql [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+(4, 13966, 11514, 2, 11511, 1,
+    'Gromsblood - Fel Creep requires Cenarion Beacon'),
+(4, 13967, 11514, 2, 11511, 1,
+    'Golden Sansam - Fel Creep requires Cenarion Beacon'),
+(4, 13968, 11514, 2, 11511, 1,
+    'Dreamfoil - Fel Creep requires Cenarion Beacon'),
+(4, 13969, 11514, 2, 11511, 1,
+    'Mountain Silversage - Fel Creep requires Cenarion Beacon'),
+(4, 13970, 11514, 2, 11511, 1,
+    'Arthas Tears - Fel Creep requires Cenarion Beacon'),
```
[the remark this answers is not in the document]
Comment (lilliandris) [reply data/sql/updates/pending_db_world/rev_1781654104088564177.sql]: Resolved this issue in latest forced commit.

**Review by Gultask — APPROVED** [review]
